src/lib/validationRules/remoteMatchingClientId/remoteMatchingClientId.ts

```diff
@@ -19,7 +19,32 @@
 // SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 //
 
-import { RemoteValidationContext, RemoteValidationRule } from "../../types";
+import {
+  RemoteValidationContext,
+  RemoteValidationRule,
+  ResultDescription,
+} from "../../types";
+
+const resultDescriptions: Record<string, ResultDescription> = {
+  missingClientId: {
+    status: "error",
+    title: "Remote Client Identifier Document has no `client_id` set",
+    description:
+      "The remote Remote Client Identifier Document must have the Client Identifier URI set to the field `client_id`.",
+  },
+  remoteLocalClientIdMismatch: {
+    status: "error",
+    title: "Remote and local Client Identifier mismatch",
+    description:
+      "The remote and the local Client Identifier URI must be equal (string equality).",
```

Comment:
What does this mean?  
it sounds like I have a local Client Identifier document and we're saying the remote URI and the local URI must match, which couldn't, right?

Reply:
Does 8ec0f9e92d6f46f30017b6b0de82ed9cdd0599bb clarifies this rule?